xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -162,6 +163,47 @@ struct EditProfileSheetCursor: CursorProtocol {
     }
 }
 
+extension Store<UserProfileDetailModel> {
+    func refresh() async {
+        guard let address = state.user?.address else {
+            return
+        }
+        
+        // Ensure updates are sent on the main thread
+        let send = { action in
+            DispatchQueue.main.async {
+                self.send(action)
+            }
```

Comment:
The new way:

```
Task { @MainActor in self.send(action) }
```

Reply:
Urgh I was looking for something like this.